src/Styleguide/Pages/Artwork/Sidebar/PartnerInfo.tsx

```diff
@@ -0,0 +1,78 @@
+import React from "react"
+import { Serif } from "@artsy/palette"
+import styled from "styled-components"
+import { space, SpaceProps } from "styled-system"
+import { Flex } from "Styleguide/Elements/Flex"
+import { Location } from "../../../../Assets/Icons/Location"
+
+export interface PartnerInfoProps {
+  artwork?: {
+    readonly collecting_institution?: string
+  }
+  partner: {
+    readonly __id: string
+    readonly name: string
+    readonly href?: string
+    readonly locations: Array<{
+      readonly city: string
+    }>
+  }
+}
+
+const PartnerInfoContainer = styled.div.attrs<SpaceProps>({})`
+  ${space};
+`
+const LocationsContainer = styled.div``
+
+export class PartnerInfo extends React.Component<PartnerInfoProps> {
+  renderPartnerName(partner) {
+    return partner.href ? (
+      <Serif size="5t" display="inline-block">
+        <a href={partner.href}>{partner.name}</a>
+      </Serif>
+    ) : (
+      <Serif size="5t" display="inline-block">
+        {partner.name}
+      </Serif>
+    )
+  }
+  renderCollectingInstitution(artwork) {
+    return <Serif size="3">{artwork.collecting_institution}</Serif>
```

Comment:
Seeing as this is a method, I’d just access `this.props.artwork` here, because it will automatically give you the right typing from the ‘props’ interface, whereas passing it as a parameter means you’d need to type it again.